components/edit-collective/Menu.js

```diff
@@ -78,6 +79,9 @@ const SECTION_LABELS = defineMessages({
     id: 'editCollective.menu.policies',
     defaultMessage: 'Policies',
   },
+  [EDIT_COLLECTIVE_SECTIONS.THANK_YOU_EMAIL]: {
+    defaultMessage: 'Thank You Email',
```

Comment:
NOTIFICATIONS Notifications

Reply:
The design mentions this as `Thank you Email`; https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=17819%3A191679 which is why I put it as such (double checked with Memo as well). But I am okay with changing this if there's consensus that `Notifications` is the correct wording. 🙂